tests/serving/test_async_flow.py

```diff
@@ -183,7 +183,7 @@ def select(self, event, available_models: list[Model]) -> Optional[list[str]]:
 
 @pytest.mark.parametrize(
     "execution_mechanism",
-    ("multiprocessing", "threading", "asyncio", "naive"),
+    ("process_pool", "dedicated_process", "thread_pool", "asyncio", "naive"),
 )
 def test_model_runner_with_selector(execution_mechanism: str):
```

Comment:
can you please add also in mlrun level logic for setting the gpu in dedicated_process mechanism

Reply:
Done.